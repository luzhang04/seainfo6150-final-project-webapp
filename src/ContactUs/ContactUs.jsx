import React from "react";
import styles from "./ContactUs.module.css";

const ContactUs = () => {
  return (
    <div className = {styles.container}>
    <article className={styles.article}>
            <h1>Contact Us</h1>
            <br />
            <p>
                <b>For customer service: </b><br />
                <br />
                <span>please email products@www.com or call 1-888-888-8888.</span>
            </p>
            <br />

            <p>
                <b>For subscription issues: </b><br />
                <br />
                <span>please email ducts@www.com.</span>
            </p>
            <br />

            <p>
                <b>For advertising-related queries: </b><br />
                <br />
                <span>please email prs@www.com.</span>
            </p>



    </article>
        </div>
  )
}

export default ContactUs;
