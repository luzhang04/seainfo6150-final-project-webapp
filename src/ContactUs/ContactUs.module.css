
.container {
  background: #edf6f9;
  border: 1px solid #edf6f9;
  font-family: 'Montserrat', Helvetica, sans-serif;
  display: grid;
  grid-template-columns: auto;
  align-items: center;
  gap: 20px;
  list-style-type: none;
  padding: 100px;
  justify-items: center;
}

.article {
  background: #edf6f9;
  border: 2px dashed #edf6f9;
  padding: 10px;
  box-sizing: border-box;
  width:50%;
}

h1 {
  color: #457b9d;
  font-weight: bold;
}
