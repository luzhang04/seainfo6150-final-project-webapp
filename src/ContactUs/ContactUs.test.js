import React from "react";
import { render } from "@testing-library/react";
import ContactUs from "./ContactUs.jsx";

describe("ContactUs tests", () => {
    it("renders correctly", () => {
      const { container } = render(<ContactUs />);
      expect(container).toMatchSnapshot();
    });
  });