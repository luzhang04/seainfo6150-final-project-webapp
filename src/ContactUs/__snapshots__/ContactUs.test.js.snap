// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <article
      class="article"
    >
      <h1>
        Contact Us
      </h1>
      <br />
      <p>
        <b>
          For customer service: 
        </b>
        <br />
        <br />
        <span>
          please email products@www.com or call 1-888-888-8888.
        </span>
      </p>
      <br />
      <p>
        <b>
          For subscription issues: 
        </b>
        <br />
        <br />
        <span>
          please email ducts@www.com.
        </span>
      </p>
      <br />
      <p>
        <b>
          For advertising-related queries: 
        </b>
        <br />
        <br />
        <span>
          please email prs@www.com.
        </span>
      </p>
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
      <br />
    </article>
  </div>
</div>
`;
